test/test_hybrid_system.jl

```diff
@@ -117,4 +127,10 @@ end
     forecast = Deterministic(name, data, resolution)
     add_time_series!(sys, thermal_unit, forecast)
     @test get_time_series(Deterministic, thermal_unit, name) isa Deterministic
+    copy_subcomponent_time_series!(h_sys, thermal_unit)
```

Comment:
This is an example of the potential problem.

Reply:
Is this something we can detect and throw a log message? I'm thinking we'd have something like:

```Julia
julia> add_time_series!(sys, thermal_unit, forecast) 
┌ Warning: thermal_name is a Subcomponent of a HybridSys Component, call `copy_subcomponent_time_series!(h_sys, subcomponent)` to enable time series access from the HybridSys
```

Reply:
The subcomponent doesn't know that it's attached to a Hybrid. The system could detect that the subcomponent is "masked" and log something about that. The system doesn't know anything about Hybrid/subcomponent, so if we wanted to log something like what you wrote, it could cause a problem if we reused the same pattern in the future.

Reply:
No worries then. If it causes strife in the future we can try to log messages based on masked status. For now, we can just make it clear in the docs.